pyspecProcScripts/third_level/process_IR.py

```diff
@@ -105,23 +138,19 @@ def process_IR(s, label='', fl=None,
     s.ft('t2')
     if fl is not None:
         fl.next('phased data -- frequency domain')
-        fl.image(as_scan_nbr(s)) 
+        fl.image(as_scan_nbr(s))
     #}}}
     #}}}
     if 'ph2' in s.dimlabels:
         s.reorder(['ph1','ph2','vd','t2'])
     else:
         s.reorder(['ph1','vd','t2'])
     #{{{Correlation Alignment
-    s.ift(['ph1','ph2'])
     fl.basename='correlation subroutine:'
     #for the following, should be modified so we can pass a mask, rather than specifying ph1 and ph2, as here
-    opt_shift,sigma = correl_align(s,indirect_dim='vd',
-            ph1_selection=signal_pathway['ph1'],ph2_selection=signal_pathway['ph2'],
+    s,opt_shift,sigma = correl_align(s,indirect_dim='vd',
+            signal_pathway=signal_pathway,
             sigma=10)
-    s.ift('t2')
-    s *= np.exp(-1j*2*pi*opt_shift*s.fromaxis('t2'))
-    s.ft('t2')
     fl.basename = None
     if fl is not None:
         fl.next(r'after correlation, $\varphi$ domain')
```

Comment:
again, I'm kind of skipping the rest of this, b/c this file doesn't seem to be something we even want --> it would be replaced by the table of integrals

Reply:
the table of integrals will be a separate PR this is again testing the correlation alignment on actual experimental data